installer/pkg/config/types.go

```diff
@@ -20,6 +20,7 @@ const (
 type Admin struct {
 	Email    string `json:"tectonic_admin_email" yaml:"email,omitempty"`
 	Password string `json:"tectonic_admin_password" yaml:"password,omitempty"`
+	SSHKey   string `json:"tectonic_admin_ssh_key,omitempty" yaml:"sshKey"`
```

Comment:
The property is optional (per your `config.tf` comment), so I think this should be `omitempty` (like the other `Admin` properties).